mathics/builtin/scoping.py

```diff
@@ -413,8 +411,7 @@ class Contexts(Builtin):
 
     ## this assignment makes sure that a definition in Global` exists
     >> x = 5;
-    >> Contexts[] // InputForm
-     = {"CombinatoricaOld`", "Global`", "ImportExport`", "Internal`", "System`", "System`Convert`B64Dump`", "System`Convert`Image`", "System`Convert`JSONDump`", "System`Convert`TableDump`", "System`Convert`TextDump`", "System`Private`", "XML`", "XML`Parser`"}
+    X> Contexts[] // InputForm
```

Comment:
And what would be the output of the test?

Reply:
My understanding [from the wiki](https://github.com/mathics/Mathics/wiki/Documentation-markup) is that `X>` just puts in something as an example. 

I don't want to specify output since that is fragile and changing over time. 

If my understanding or the wiki is wrong please suggest a change or better just fix. Thanks.

Reply:
Sorry, you are right. I just forgot it. Thanks!